symfit/core/minimizers.py

```diff
@@ -465,7 +466,76 @@ class Powell(ScipyMinimize, BaseMinimizer):
     """
     Wrapper around :func:`scipy.optimize.minimize`'s Powell algorithm.
     """
+    
 
+class TrustConstr(ScipyConstrainedMinimize, GradientMinimizer, BoundedMinimizer):
+    """
+    Wrapper around :func:`scipy.optimize.minimize`'s Trust-Constr algorithm.
+    """
+    def __init__(self, *args, **kwargs):
+        super(TrustConstr, self).__init__(*args, **kwargs)
+        # We have to break DRY because you cannot inherit from both
+        # ScipyConstrainedMinimize and ScipyGradientMinimize. So SLQSP is a
+        # special case. This is the same code as in ScipyGradientMinimize.
+        self.wrapped_jacobian = self.list2kwargs(self.wrapped_jacobian)
+
+    @keywordonly(jacobian=None, hess=None, options=None)
```

Comment:
I'd use `hessian` in full, in keeping with using `jacobian` instead of `jac` like scipy does.